src/cache.js

```diff
@@ -33,7 +33,7 @@ export function getTmp () {
  * @return {WriteStream} - Write Stream
  */
 export function write () {
-	return fs.WriteStream(getTmp())
+	return fs.createWriteStream(getTmp())
```

Comment:
Were encountering the eslint [new-cap](http://eslint.org/docs/rules/new-cap) rule: `A function with a name starting with an uppercase letter should only be used as a constructor.`